core/account-id/src/lib.rs

```diff
@@ -358,12 +358,6 @@ impl fmt::Display for AccountId {
     }
 }
 
-impl From<AccountId> for String {
-    fn from(account_id: AccountId) -> Self {
-        account_id.0.into_string()
-    }
-}
-
```

Comment:
This one I think is intentional, as it avoid copy: `.to_string()` will give the same result, but with an allocation. Probably make sense to add a commet on why we need the two impls.